subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.h

```diff
@@ -38,6 +38,9 @@ struct lll_scan {
 
 	uint16_t interval;
 	uint32_t ticks_window;
+	uint32_t period;
```

Comment:
move fields to ll_scan_set, as lll_scan member are the ones used/modified by LLL.

Reply:
Fixed